src/pkg/packager/creator/normal.go

```diff
@@ -186,11 +187,15 @@ func (pc *PackageCreator) Assemble(dst *layout.PackagePaths, components []types.
 			}
 
 			pulled, err = imgConfig.PullAll()
+			if errors.Is(err, lang.ErrUnsupportedImageType) {
```

Comment:
I think it would be good to provide some context by providing this error

Reply:
Sorry a bit confused by this suggestion. By context do you mean comments in code / logging to user / or adding values to the actual go context object?

Reply:
I was unclear, I would say "logging to user", but I'll never tell you not to add comments ;)

Reply:
Ah yeah, the reason I don't log it here is because the error gets logged to the user during the retry function

Reply:
After typing it, I went "this probably gets outputted somewhere not here", and to be transparent I just didn't look. 🙃